modules/c/nitf/source/ImageIO.c

```diff
@@ -3922,6 +3922,11 @@ nitf_ImageIO_getBlockingInfo(nitf_ImageIO * image,
             img->dataLength - img->maskHeader.imageDataOffset,
             &(img->blockInfo), img->blockMask, error) )
         {
+            // On failure, the decompressor frees the decompressionControl
```

Comment:
Since this is C, need to use `/* */`

Reply:
Fixed. Also updated the header comments to clarify what happens.